schemas/json/aas.json

```
@@ -683,7 +683,7 @@
                   "pattern": "^([\\t\\n\\r -\ud7ff\ue000-\ufffd]|\\ud800[\\udc00-\\udfff]|[\\ud801-\\udbfe][\\udc00-\\udfff]|\\udbff[\\udc00-\\udfff])*$"
```

```suggestion
                  "pattern": "^([\\t\\n\\r-\\ud7ff\\ue000-\\ufffd]|\\ud800[\\udc00-\\udfff]|[\\ud801-\\udbfe][\\udc00-\\udfff]|\\udbff[\\udc00-\\udfff])*$"
```
Also for all other occurrences of this pattern.

---

Good eye, I'll adapt this and regenerate the schema files

---

Actually those appear to be two problems here in one. 
- One is that the Path_type regex is actually still in here, which is relatively easy to fix
- The second one however is this missing backslash. @mristin This appears to be a bug in aas-core-codegen, since the regex looks fine to me in [aas-core-meta](https://github.com/aas-core-works/aas-core-meta/blob/6d5411bf9f5ef0ccc1ed956c63c2eca50711137f/aas_core_meta/v3.py#L365)

---

@s-heppner @sebbader-sap I'm working on it today. It's amazing when such strange & obvious bugs slip so many eyes!

---

Just for future reference: this is actually not a bug. As long as a character is in the Basic Multilingual Plane (BMP, <0xFFFF), the JSON Schema engines will automatically understand the character. Hence, it does not matter whether you write:
```
"pattern": "\ud7ff"
```

... or:
```
"pattern": "\\ud7ff"
```

The first pattern is given with a character as-is, the second pattern escapes the character. This matters when, say, you print out the violation message (most engines indicate the pattern violates). The first message will contain the pattern with the character as-is and the second message will be shown with the pattern escaped.

This still needs to be fixed for consistency.